client/src/Components/RepoBrowser/RepositoryAPITile.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022 Microsoft Corporation
+// Copyright (c) 2023 Marc Lichtman
+// Licensed under the MIT License
+
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useAppDispatch } from '@/Store/hooks';
+import { CLIENT_TYPE_API } from '@/api/Models';
+
+const RepositoryAPITile = (props) => {
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const { item } = props;
+
+  const { type, name, account, container, imageURL, description } = item;
+  const [isDisabled, setIsDisabled] = useState(false);
+  const [writeableBool, setWriteableBool] = useState<any>(true);
+
+  useEffect(() => {
+    const writeable = true; // boolean
```

Comment:
This shouldn't be needed for the api should it?

Reply:
I understand the R/W indicator is going away now. It was retained to allow indication once the API had sas token capability